pkg/prometheus/promcfg.go

```diff
@@ -1492,7 +1523,7 @@ func (cg *ConfigGenerator) generateRemoteWriteConfig(version semver.Version, p *
 		}
 
 		if spec.BasicAuth != nil {
-			if s, ok := basicAuthSecrets[fmt.Sprintf("remoteWrite/%d", i)]; ok {
+			if s, ok := basicAuthSecrets[fmt.Sprintf("remoteRead/%d", i)]; ok {
```

Comment:
Any reason for this change? This should be remoteWrite considering that we are generating the remote write configuration.

Reply:
This seems to be a typo, good catch.